src/components/common/FormBottomBar.tsx

```diff
@@ -39,20 +39,20 @@ export interface Props {
   cancelID: string;
   saveID: string;
   onCancel: () => void;
-  onSave: () => void;
   cancelButtonText?: string;
   saveButtonText?: string;
   saveDisabled?: boolean;
+  handleSave: () => void;
```

Comment:
this can still be onSave.. no need to push the name change from the other place in here